app/services/cancel_reservation.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class CancelReservation
+  def initialize(reservation)
+    @reservation = reservation
+  end
+
+  def call
+    return false unless booked?
```

Comment:
```suggestion
    return unless booked?
```

Suggested syntax is just a syntax sugar. If you do not specify a value (fe. `false`) then method will return `nil`. 
In case of same conditions `nil` will cast to `false` boolean value, so in this case it should be the same.